### DISCUSSION TITLE
Jupiter Moons not lining up with what I see?

### DISCUSSION THREAD

**Discussion by JGSmith12** [Stellarium/stellarium#2993] [Q&A]
Hello,

A couple of nights ago I was looking at Jupiter. I took a basic picture with my phone through the eyepiece and see 3 of Jupiter's moons. I then took a screenshot from my Stellarium iPhone app to compare them. They are significantly different. The app's location and date/time is set correctly.

![moons](https://user-images.githubusercontent.com/122671423/212466411-33a11c1f-f72f-46e5-bba2-15d18d6fb597.png)

Does anybody know why this could be so different?

Thank you!

Jason

**Comment by gzotti** [comment]
What is different?

**Reply by JGSmith12** [reply]
The difference being the rotation. But this might be just my inexperience in looking through a telescope. There is about an 80 degree difference in the angle that the moons are at.

**Comment by 10110111** [comment]
Looks similar enough IMO:

![212466411-33a11c1f-f72f-46e5-bba2-15d18d6fb597](https://user-images.githubusercontent.com/6376882/212467225-b79befce-50ed-47e0-a303-759f9ed9c746.png)

**Comment by JGSmith12** [comment]
Okay, when you turn it like that, it looks perfect!

I guess the issue that I have is why it would be rotated. Could that have to do with the angle that the telescope was turned?

**Reply by 10110111** [reply]
I don't think the telescope orientation would rotate the image by such weird angles (it may rotate by 180° though). But the orientation you get on the photo depends on how you orient the camera with respect to the eyepiece. Stellarium normally shows everything in such a way that "up" direction points to the zenith.
